src/features/tools/change-bounds-tool.ts

```diff
@@ -164,11 +176,23 @@ export class ChangeBoundsListener extends MouseListener implements SelectionList
         } else {
             // Move
             const newBounds: ElementAndBounds[] = [];
-            forEachElement(target, isNonRoutableSelectedMovableBoundsAware, element =>
-                this.createElementAndBounds(element).forEach(bounds => newBounds.push(bounds)));
+            const newRoutingPoints: ElementAndRoutingPoints[] = [];
+            forEachElement(target, isNonRoutableSelectedMovableBoundsAware, element => {
+                this.createElementAndBounds(element).forEach(bounds => newBounds.push(bounds));
+                //  If client routing is enabled -> delegate routingpoints of connected edges to server
+                if (this.tool.edgeRouterRegistry && element instanceof SConnectableElement) {
+                    element.incomingEdges.map(toElementAndRoutingPoints).forEach(ear => newRoutingPoints.push(ear));
+                    element.outgoingEdges.map(toElementAndRoutingPoints).forEach(ear => newRoutingPoints.push(ear));
+                }
+            }
```

Comment:
Just a non-blocking minor: If it isn't too much hassle, I'd put the `);` at the end of line 187:
`});`